server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -154,21 +249,17 @@ async fn get_messageattempt(
 async fn resend_webhook(
     Extension(ref db): Extension<DatabaseConnection>,
     Extension(queue_tx): Extension<TaskQueueProducer>,
-    Path((_app_id, msg_id, endpoint_id)): Path<(
-        ApplicationIdOrUid,
-        MessageIdOrUid,
-        EndpointIdOrUid,
-    )>,
+    Path((_app_id, msg_id, endp_id)): Path<(ApplicationIdOrUid, MessageIdOrUid, EndpointIdOrUid)>,
```

Comment:
These changes are unrelated. It's not the end of the world this time, but please try to avoid renaming variables in a PR without a good reason.